### DISCUSSION TITLE
Fixed table height

### DISCUSSION THREAD

**Discussion by manuelpoelzl** [fomantic/Fomantic-UI#535] [Collections (form, grid, menu, message, table, ...)] [answered]
Is there a possibility to set a fixed table height like 600px or something inside a grid column?

It seems that every height property i set is ignored

thanks!

**Comment by lubber-de** [comment, marked as answer]
Here is something i prepared for a possible upcoming PR
`scrolling content table`
It's not perfect yet, but maybe it helps you for now:
https://jsfiddle.net/49xgjau8/1/

Change the `height` in the CSS Part for `.mytable > tbody` to whatever you wish

![image](https://user-images.githubusercontent.com/18379884/53732981-d1044f00-3e7f-11e9-9dda-27fec4025fff.png)

**Comment by manuelpoelzl** [comment]
Works like a charm!
Thank you very much!

**Reply by lubber-de** [reply]
`scrolling table` now available by #2134

**Reply by manuelpoelzl** [reply]
Thank you for notifying me!
